setup.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import unicode_literals
```

Comment:
This is the standard Sideboard-provided ``setup.py`` file.  If we had any extra dependencies, this is what would install them into Sideboard's virtualenv.